src/structures/GuildMember.js

```diff
@@ -81,6 +81,17 @@ class GuildMember extends Base {
     return clone;
   }
 
+  /**
+   * The Message object of the last message sent by the member in their guild, if one was sent
+   * @type {?Message}
+   * @readonly
+   */
+  get lastMessage() {
+    const channel = this.guild.channels.get(this.lastMessageChannelID);
+    if (!channel) return undefined;
```

Comment:
shouldn't this return `null` instead to stay consistent with other getters/methods?